src/components/HathorHeader.js

```diff
@@ -75,7 +76,7 @@ const HathorHeader = (props) => {
 
 const styles = StyleSheet.create({
   wrapper: {
-    height: 56,
+    height: HEADER_HEIGHT,
```

Comment:
The other PR you stop using this constant

Reply:
That's right: while this PR was already under review I came up with a better organization for this on 326.
This variable will be short lived on the `constants` file.